components/eventCard/EventCard.tsx

```diff
@@ -1,104 +1,98 @@
 import React from "react"
 import { Image, StyleSheet, Text, View } from "react-native"
 import { CurrentUserEvent, isHostingEvent } from "@lib/events/Event"
-import { Divider } from "react-native-elements"
-import { Shadow } from "react-native-shadow-2"
-import tinycolor from "tinycolor2"
-import { placemarkToFormattedAddress } from "@lib/location"
-import { compactFormatMiles } from "@lib/DistanceFormatting"
-import { MaterialCommunityIcon } from "../common/Icons"
+import { placemarkToAbbreviatedAddress } from "@lib/location"
+import { Ionicons } from "@expo/vector-icons"
+import { useFonts } from "expo-font"
 import MenuDropdown from "./MenuDropdown"
 
 export type EventCardProps = {
   event: CurrentUserEvent
 }
 
 export const EventCard = ({ event }: EventCardProps) => {
-  const lightEventColor = tinycolor(event.color).lighten(25).toString()
+  const hexAlpha = "4D"
+  const lightEventColor = event.color + hexAlpha
+
+  const [fontsLoaded] = useFonts({
+    "Open-Sans": require("../../assets/fonts/OpenSans-Regular.ttf"),
+    "Open-Sans-Bold": require("../../assets/fonts/OpenSans-Bold.ttf")
+  })
+
+  if (!fontsLoaded) {
+    return null
+  }
 
   return (
-    <Shadow
-      distance={16}
-      offset={[-2, 4]}
-      shadowViewProps={{ style: { opacity: 0.25 } }}
-      startColor={lightEventColor}
-      stretch={true}
-    >
-      <View style={styles.container}>
-        <View style={[styles.topRow, styles.flexRow]}>
-          <Image
-            style={[styles.image, styles.iconMargin]}
-            source={require("../../assets/icon.png")}
-            accessibilityLabel="profile picture"
-          />
+    <View style={styles.container}>
+      <View style={[styles.topRow, styles.flexRow]}>
+        <Image
+          style={[styles.image, styles.iconMargin]}
+          source={require("../../assets/icon.png")}
+          accessibilityLabel="profile picture"
```

Comment:
Ty for accessibility label. We'll need a separate component for the profile image at least since I'm sure we'll reuse that a lot in different screens.